### ISSUE TITLE
[RNWin32] Turn on enableFocusRing by default on View

### ISSUE THREAD

**Issue by rurikoaraki** [microsoft/react-native-windows#9111]
### Summary

Switch enableFocusRing to be true by default (requires a native change. This issue is for tracking and making intent of behavior public, there should be no changes in the repo itself to implement this behavior.)

### Motivation

RNWin32 has operated with mostly having clients implement their own focus UI through JS. With adding enableFocusRing, in order to make sure that the new prop doesn't cause any issues on introduction, we are starting with the default to false, but want to transition to turning it on by default to align with other platforms long term. This issue is to track this intent.

**Comment by NickGerleman** [comment]
@rurikoaraki is this completed?

**Comment by rurikoaraki** [comment]
> @rurikoaraki is this completed?

@NickGerleman No. I have an item on my plate for it, but I've been swamped with other work and this isn't very high on my priority list.
